client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -1,11 +1,27 @@
 <template>
     <DatasetProvider :id="hda_id" v-slot="{ item: dataset, loading }">
         <div v-if="!loading">
-            <h3>Dataset Information</h3>
+            <tr>
+                <td>
+                    <h3>Dataset Information</h3>
+                </td>
+                <td>
+                    <b-button
+                        class="edit-button"
```

Comment:
@simonbray there's more, this for instance probably doesn't make sense now that we display the dataset component in line, which already has the edit button

Reply:
I removed the edit button as well as the job rerun button; both are already present in the newly embedded dataset / job components.